Client/src/components/common/LabeledInput/index.css

```diff
@@ -0,0 +1,3 @@
+.LabeledInput{
```

Comment:
`.labeledInput` is better

Take a look at this [https://www.quora.com/What-is-the-difference-between-Pascal-Case-and-Camel-Case](url)